build/tasks/build.js

```diff
@@ -81,7 +84,7 @@ module.exports = function( grunt ) {
 
 			// Remove define wrappers, closure ends, and empty declarations
 			contents = contents
-				.replace( /define\([^{]*?{/, "" )
+				.replace( /define\([^{]*?{\s*(?:"use strict";)?/, "" )
```

Comment:
Given recent talk of changing code style, the directive-matching subexpression should probably be more general, e.g. accepting a trailing [line terminator](http://www.ecma-international.org/ecma-262/6.0/#sec-line-terminators) instead of semicolon and stripping [whitespace](http://www.ecma-international.org/ecma-262/6.0/#sec-white-space) and comments:
```
// was: (?:"use strict";)
(?:("|')use strict\1[\t\v\f \xA0\uFEFF]*(?:\/\*[\w\W]*?\*\/[\t\v\f \xA0\uFEFF]*)*(?:\/\/.*)?[\n\r;])
```